lib/model/migrations/20220628-01-enable-pg-stat-statements.js

```diff
@@ -0,0 +1,16 @@
+// Copyright 2022 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const { sql } = require('slonik');
```

Comment:
We actually use Knex rather than Slonik for migrations! We've mostly moved away from Knex to Slonik, but migrations still use Knex.

Reply:
...and also this isn't even used!  :facepalm: :facepalm: